k8s.io/OWNERS

```diff
@@ -5,6 +5,7 @@ options:
 approvers:
 - ixdy
 - thockin
+- dims
```

Comment:
We should get parity between approvers and this group https://github.com/kubernetes/k8s.io/blob/390481ba911059c23a2739cb56bccf8be3ea183b/groups/groups.yaml#L543-L554

```suggestion
- aojea
- bentheelder
- cblecker
- dims
- spiffxp
```

(you'll also need to remove ixdy)

Can you update this to add yourself to the group? Add thockin and myself (@google.com) if it's not too much trouble?